### PR TITLE
Doc/655 documentation landing page

### PR DESCRIPTION
**Pull request by devtobi**
**Description**
- Added custom LHM theme
- Updated vitepress configuration
- Added landing page
- Added Prettier to linting toolchain

**Reference**
Issues #655

## Release Notes

- **New Features**
  - Introduced a custom theme for VitePress with a new logo in the navigation bar.
  - Updated documentation homepage with new sections and links for improved navigation.

- **Bug Fixes**
  - Enhanced JSON formatting in configuration files to ensure consistency.

- **Documentation**
  - Updated Vitepress configuration for better structure and user guidance.
  - Improved home page layout and feature descriptions for clarity.

- **Chores**
  - Integrated Prettier for code formatting in the linting process.
  - Added new entries to `.prettierignore` to streamline formatting tasks.
  - Added a new dependency for Prettier code formatting.

### CODE REVIEW

File: docs/.vitepress/theme/style.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+:root {
+  --muc-logo-filter: invert(0);
```
Comment (simonhir) [inline docs/.vitepress/theme/style.css:2]: Does that work with the svg image?

Comment (devtobi) [reply docs/.vitepress/theme/style.css:2]: yes

**Review by simonhir — DISMISSED** [review]
LGTM

File: docs/.vitepress/theme/style.css [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+}
+
+/* Enables footer on all pages instead of only the home page */
+footer.VPFooter {
+  display: block !important;
+  z-index: 100 !important;
+}
+
+.VPNav div.logo a img {
+  min-width: 169px;
```
Comment (simonhir) [inline docs/.vitepress/theme/style.css:12]: For what is that good?

**Review by simonhir — APPROVED** [review]
LGTM

**Pull request merged**